README.md

```diff
@@ -72,6 +72,18 @@ The `stack_rm` orb removes a stack and its configuration..
 | force | boolean | false | Whether or not to force deletion of the stack, leaving behind any resources managed by the stack. | 
 | working_directory | string | . | The relative working directory to run `pulumi` from. | 
 
+### pulumi/stack_output
+
+The `stack_output` sets a stack's output property to an environment variable.
+
+| Parameter         | type    | default     | description    |
+|-------------------|---------|-------------|----------------|
+| stack           | string  | (none)      | Name of the stack to remove. |
+| property_name           | string  | (none)      | Property name of the stack output to set to the environment variable. |
```

Comment:
Perhaps:

~~Property name of the stack output to set to the environment variable.~~
name of the output property to read